packages/mobile/src/verify/VerificationLoadingScreen.tsx

```diff
@@ -123,8 +126,8 @@ class VerificationLoadingScreen extends React.Component<Props> {
           <View style={styles.buttonCancelContainer}>
             <CancelButton onCancel={this.onCancel} />
           </View>
+          <DevSkipButton nextScreen={Screens.VerificationInterstitialScreen} />
```

Comment:
I can't remember if this is my bug or someone else's but this screen shouldn't have a DevSkipButton. It won't work as expected with verification ongoing in the background. If devs want to skip they should just choose skip on the education screen

Reply:
Makes sense yes, removing then 👍